src/biomappings/resources/predictions.tsv

```diff
@@ -1440,7 +1440,6 @@ mesh	C110058	PIBF1 protein, human	skos:exactMatch	uniprot	Q8WXW3	PIBF1	lexical	h
 mesh	C464442	SIGLEC11 protein, human	skos:exactMatch	uniprot	Q96RL6	SIGLEC11	lexical	https://github.com/biomappings/biomappings/blob/a840cf/scripts/generate_mesh_uniprot_mappings.py
 mesh	C402206	SIGLEC7 protein, human	skos:exactMatch	uniprot	Q9Y286	SIGLEC7	lexical	https://github.com/biomappings/biomappings/blob/a840cf/scripts/generate_mesh_uniprot_mappings.py
 mesh	C491094	SIM1 protein, human	skos:exactMatch	uniprot	P81133	SIM1	lexical	https://github.com/biomappings/biomappings/blob/a840cf/scripts/generate_mesh_uniprot_mappings.py
-mesh	C501921	SIRPB1 protein, human	skos:exactMatch	uniprot	O00241, Q5TFQ8	SIRPB1	lexical	https://github.com/biomappings/biomappings/blob/a840cf/scripts/generate_mesh_uniprot_mappings.py
```

Comment:
just double checking - you do mean to remove these predictions?

Maybe we should store non-trivial "negative" mappings as you find bad ones

Reply:
I really wanted to remove these. These are not quite "negative" mappings, rather, they are not one-to-one, because e.g., the SIRPB1 gene corresponds to two UniProt entries.

Reply:
I did start introducing explicit negative mappings in #6 using `^` as a prefix to the relation.